Move a level of nesting from all student yaml

ryansb opened the pull request:
Used: `find ./scripts/people -type f -name '*yaml' -exec sed -i -e 's/^- //'  -e 's/^  //' -e 's/\s\+$//' {} \;` to remove a nesting level. 

Don't merge this until I've fixed uses of student yaml not to expect nesting.

liam-middlebrook committed:
Added more things to participant page
* GitHub Badge
* Biography Text
* Twitter feed

liam-middlebrook committed:
Added coderwall 'chevo list to participant page

liam-middlebrook committed:
Bio header should only appear if the user has a bio

liam-middlebrook committed:
Updated loothelion.yaml

ryansb committed:
Move a level of nesting from all student yaml

ryansb commented:
Review protip: use https://github.com/decause/hflossk/pull/421/files?w=1 to look at nonwhitespace changes.

liam-middlebrook commented:
Should there be a change in site.py? 

https://github.com/decause/hflossk/blob/develop/hflossk/site.py#L100

I believe that this line would cause issues with the new YAML standard

liam-middlebrook committed:
Updated site.py and participants.py to use new YAML format

liam-middlebrook committed:
Merge branch 'feature/participant_page' into hwListYAML

Conflicts:
	scripts/people/2014/spring/loothelion.yaml

liam-middlebrook committed:
Updated YAMLS to use a list for all homework/blogposts

liam-middlebrook committed:
Updated participants listing to only show red assignments for students
who are in the current term of HFOSS

Legacy students will have all of their assignments listed

ryansb committed:
Merge pull request #422 from liam-middlebrook/fixYaml

Updated site.py and participants.py to use new YAML format

ryansb committed:
Merge pull request #423 from liam-middlebrook/hwListYAML

YAML HW List
